### PR TITLE
feat: windows binary support

### PR DESCRIPTION
**Pull request by jr0d**
https://jira.d2iq.com/browse/D2IQ-70048

### CODE REVIEW

File: asyncauth.go [hunk 1]
```diff
@@ -23,7 +23,8 @@ import (
 const (
 	HmacTTL            = 300
 	downloadPath       = "static/downloads/"
-	binaryName         = "konvoy-async-auth"
+	binaryName         = "konvoy-async-plugin"
+	binaryNameWin32    = "konvoy-async-plugin.exe"
```
Comment (shaneutt) [inline asyncauth.go]: Should we start a habit of using `Windows` as opposed to `Win32`, just to make sure our nomenclature is consistent? We're only supporting amd64 builds with these changes anyway, right?

File: asyncauth.go [hunk 2]
```diff
@@ -147,9 +151,9 @@ func renderPluginInstructions(w http.ResponseWriter, data TemplateData) error {
 	return nil
 }
 
-func getDownloadURL(url, platform, version string) string {
+func getDownloadURL(url, platform, version, binary string) string {
 	// TODO: make this more readable
-	return fmt.Sprintf("%s%s%s/%s_%s/%s", url, downloadPath, platform, binaryName, version, binaryName)
+	return fmt.Sprintf("%s%s%s/%s_%s/%s", url, downloadPath, platform, "konvoy-async-auth", version, binary)
```
Comment (shaneutt) [inline asyncauth.go]: Perhaps we should put what is now the "binary base name" into a constant.

**Review by shaneutt — APPROVED** [review]
LGTM :+1: 

I have only non-blocking minor comments.

File: asyncauth.go (hunk 1 above)
Comment (jr0d) [reply asyncauth.go]: win32 as in the API.. force of habit.

Comment (shaneutt) [reply asyncauth.go]: Ok, like I said non blocking, just a thought.

File: asyncauth.go (hunk 2 above)
Comment (jr0d) [reply asyncauth.go]: :+1:

**Pull request merged**